presto-jdbc/src/main/java/com/facebook/presto/jdbc/QueryExecutor.java

```diff
@@ -82,12 +82,17 @@ protected void finalize()
         close();
     }
 
-    static QueryExecutor create(String userAgent, JettyIoPool jettyIoPool)
+    static HttpClientConfig baseClientConfig()
```

Comment:
Move this method to `PrestoDriver` and make it `private`. It doesn't seem to belong here.